api.md

```diff
@@ -3068,7 +3068,7 @@ proc_schema       | _timescaledb_internal
 proc_name         | policy_refresh_continuous_aggregate
 owner             | postgres
 scheduled         | t
-config            | {"end_interval": "20 days", "start_interval": "10 
+config            | {"end_offset": "20 days", "start_offset": "10 
```

Comment:
Hmm, this config doesn't make sense. I guess end_offset should be smaller than start_offset?